templates/bundles/DemosPlanCoreBundle/DemosPlanMap/map_admin.html.twig

```diff
@@ -6,7 +6,9 @@
         heading: 'drawing.mapsection.define'|trans
     } %}
 
-    <map-admin procedure-id="{{ procedure }}">
+    <map-admin
+        :is-master="{{ master | json_encode() }}"
```

Comment:
Just a question because I don't know it: when do we use `|json_encode()` and when do we use `|json_encode|e('js')` (or is it the same?)? Is it needed here? I'd have used `Boolean({{ master }})` :thinking:

Reply:
I'm not sure either. Twig syntax is always confusing for me. Actually I did not know `Boolean({{ master }})`, but I think it is better, thanks! Done here [d35b132](https://github.com/demos-europe/demosplan-core/pull/3738/commits/d35b132a4d076af6c2e082dc499cd07de68c3c39)